drafts/two-way-binding-signals-and-query-params.md

```diff
@@ -0,0 +1,754 @@
+---
+title: Two-way binding between Signals and Query Params
+subtitle: "Pushing the limits of signals while rediscovering Angular concepts"
+domain: software-engineering-corner.zuehlke.com
+tags: signals, web-development, javascript, typescript, frontend-development, reactivity, explained, angular
+cover: https://cdn.hashnode.com/res/hashnode/image/upload/v1710531988213/wpYOexURa.jpg?auto=format
+publishAs: juliocastrodev
+hideFromHashnodeCommunity: false
+saveAsDraft: true
+---
+
+Signals are the hot topic of today. Pretty much every major framework (except `React`...) is introducing them in one way or another. This is surprisingly also the case of Angular, which is making huge efforts to improve a lot of the pain points we've been experiencing for too many years. A few examples of this include standalone components, a new build system leveraging esbuild and our protagonist of today: signals. 
```

Comment:
**nitpick:** I wouldn't make a framework/library name in "code style" ;) same with "RxJS" in the rest of the article
```suggestion
Signals are the hot topic of today. Pretty much every major framework (except React...) is introducing them in one way or another. This is surprisingly also the case of Angular, which is making huge efforts to improve a lot of the pain points we've been experiencing for too many years. A few examples of this include standalone components, a new build system leveraging esbuild and our protagonist of today: signals. 
```